go.mod

```diff
@@ -4,32 +4,26 @@ go 1.15
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.1.5-0.20190629210212-52e137b8d003
-	github.com/klauspost/compress v1.7.5 // indirect
-	github.com/klauspost/cpuid v1.2.1 // indirect
-	github.com/operator-framework/operator-sdk v0.16.0
-	github.com/prometheus/client_golang v1.4.1
-	github.com/sirupsen/logrus v1.4.2
+	github.com/operator-framework/operator-sdk v0.18.2
+	github.com/prometheus/client_golang v1.11.0
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.7.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.17.3
+	k8s.io/api v0.18.19
+	k8s.io/apimachinery v0.18.19
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
-	k8s.io/kubernetes v1.16.2
-	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f
-	sigs.k8s.io/controller-runtime v0.4.0
-	sigs.k8s.io/controller-tools v0.2.4
+	k8s.io/kubectl v0.18.6
+	k8s.io/kubernetes v1.18.6
+	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
+	sigs.k8s.io/controller-runtime v0.6.4
+	sigs.k8s.io/controller-tools v0.3.0
 	sigs.k8s.io/kustomize v2.0.3+incompatible
-	sigs.k8s.io/yaml v1.1.0
-	vitess.io/vitess v0.0.0-20210504115542-f7304cd1893a
+	sigs.k8s.io/yaml v1.2.0
+	vitess.io/vitess v0.10.3-0.20210914162054-9df4dc751764
```

Comment:
Why is this `0.10.3`? I'd have expected `0.11.0`

Reply:
`go get vitess.io/vitess@HEAD` is what I ran

`go get vitess.io/vitess@main` got me `v0.11.1` but then that looks like not `HEAD` because it didn't have updated k8s client in it

Reply:
that makes sense. We might have to fix something on the vitess side so that the number on `@HEAD` gets bumped.